model/jpa/pom.xml

```diff
@@ -84,7 +90,14 @@
                     <groupId>com.h2database</groupId>
                     <artifactId>h2</artifactId>
                     <scope>test</scope>
-                </dependency>                
+                </dependency>
+                
+                <dependency>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-log4j12</artifactId>
+                    <version>1.6.1</version>
```

Comment:
regarding adding slf4j, please check https://github.com/aerogear/aerogear-unifiedpush-server/pull/508 

@sebastienblanc is also working on logger bits